decidim-admin/app/packs/src/decidim/admin/import_guidance.js

```diff
@@ -1,28 +1,61 @@
 $(() => {
+  const $creatorDropdownWrapper = $("label[for='import_creator']");
   const $creatorSelect = $("#import_creator");
   const $creatorGuidances = $(".creator-guidances").find(".guidance");
 
-  const showGuidance = (text) => {
-    const formatted = text.replace(/\s/g, "").toLocaleLowerCase();
-    $.each($creatorGuidances, (_index, currentValue) => {
-      if (currentValue.className.includes(formatted)) {
-        const elem = $(currentValue)
-        elem.show();
+  const classSuffix = (rubyClass) => {
+    return rubyClass.split("::").slice(-1)[0].toLowerCase();
+  }
+
+  const showTitle = (suffix) => {
+    $(`#${suffix}`).show()
+    $(`#${suffix}`).siblings().hide()
+  }
+
+  const showGuidance = (suffix) => {
+    const $elem = $(`.guidance.creator-${suffix}`)
+    $elem.show()
+    $elem.siblings().hide()
+  }
+
+  const detectCreator = (queryString) => {
```

Comment:
Could we move the creator detection logic to the backend?

So, if the creator is provided in the URL parameter (and is a known creator), we don't show the dropdown at all in the backend.

Reply:
Yes